packages/manager/src/features/Events/FormattedEventMessage.tsx

```diff
@@ -0,0 +1,66 @@
+import { styled } from '@mui/material/styles';
+import * as React from 'react';
+
+import { Link } from 'src/components/Link';
+
+interface MessageLinkEntity {
+  message: null | string;
+}
+
+/**
+ * Renders a message with inline code blocks.
+ * Meant to be used in the context of an event message.
+ * This component is only used to render {e.message} as JSX in order to:
+ *  - render ticks as <pre>.
+ *  - render "contact support" strings as <Link>.
+ */
+export const FormattedEventMessage = (props: MessageLinkEntity) => {
+  const { message } = props;
+
+  if (!message) {
+    return null;
+  }
+
+  return formatMessage(message);
+};
+
+const formatMessage = (message: string): JSX.Element => {
+  const parts = message.split(/(`[^`]*`)/g);
+  const supportLinkMatch = /(contact support)/i;
+
+  return (
+    <>
+      {parts.map((part, i) => {
+        let formattedPart: JSX.Element | string = part;
+
+        if (part.startsWith('`') && part.endsWith('`')) {
+          formattedPart = (
+            <StyledPre key={`${i}-${part}`}>{part.slice(1, -1)}</StyledPre>
+          );
+        }
+
+        if (part.match(supportLinkMatch)) {
+          const [before, linkText, after] = part.split(supportLinkMatch);
+
+          formattedPart = (
+            <span key={`${i}-${part}`}>
+              {before}
+              <Link to="/support/tickets">{linkText}</Link>
```

Comment:
```suggestion
              <SupportLink text={linkText} />
```
We can use the `SupportLink` component for this. It will link directly to a blank support ticket dialog, instead of just the support landing page.

Reply:
I agree with making use of `<SupportLink />` here

Reply:
great call! done. 

unrelated, this should just be a route, i dunno why we have a component for this...